src/GraphErrorHandler.ts

```diff
@@ -84,19 +76,21 @@ export class GraphErrorHandler {
 	 * @static
 	 * @async
 	 * To get the GraphError object
+	 * Reference - https://docs.microsoft.com/en-us/graph/errors
 	 * @param {any} [error = null] - The error returned by graph service or some native error
 	 * @param {number} [statusCode = -1] - The status code of the response
 	 * @param {GraphRequestCallback} [callback] - The graph request callback function
 	 * @returns A promise that resolves to GraphError instance
 	 */
 	public static async getError(error: any = null, statusCode: number = -1, callback?: GraphRequestCallback): Promise<GraphError> {
```

Comment:
type use

Reply:
Same reason as above.  We need to consider any custom error that consumers currently can be passing. Strict type use would be a breaking change.